src/misc/00_intro.rst

```diff
@@ -6,6 +6,12 @@ Glossary
 
 .. glossary::
 
+Conda package
+     **C**\ onda **P**\ ackage.A conda package is a .conda file that contains system-level libraries,Python or other modules,executable programs and other components,metadata under the info/ directory and a collection of files that are installed directly into an install prefix. `Learn More <https://en.wikipedia.org/wiki/Conda_(package_manager)>`__.
```

Comment:
There's some spacing issues here. Needs a space between the period and "A" at the beginning of the first sentence, and after each comma.